src/chapter9a/Person.java

```diff
@@ -0,0 +1,35 @@
+package chapter9a;
+
+public class Person {
+    String name;
```

Comment:
all the parameters are default. Please, make sure they are private or protected. There is no sense to create getters and setters if they are default.